census/core.py

```diff
@@ -158,6 +197,10 @@ def get(self, fields, geo, year=None, **kwargs):
 
     @retry_on_transient_error
     def query(self, fields, geo, year=None, **kwargs):
+        cast_nulls = kwargs.get('cast_nulls', True)
+        if cast_nulls not in [True, False]:
+            raise CensusException('cast_nulls argument must be True or False')
```

Comment:
I'm not totally confident in `cast_nulls` as the best name for the configuration argument that adjusts this behavior. In particular, it doesn't communicate that the alternative is to raise an error. Still, it was the best I had.